src/ints/mouse.cpp

```diff
@@ -1099,6 +1190,135 @@ Bitu UIR_Handler(void) {
 	return CBRET_NONE;
 }
 
+// ***************************************************************************
+// External notifications
+// ***************************************************************************
+
+void Mouse_SetSensitivity(int32_t sensitivity_x, int32_t sensitivity_y) {
+    static constexpr float MIN = 0.01f;
+    static constexpr float MAX = 100.0f;
+
+    mouse_config.sensitivity_x = std::clamp(sensitivity_x / 100.0f, -MAX, MAX);
+    if (!std::signbit(mouse_config.sensitivity_x))
+        mouse_config.sensitivity_x = std::max(mouse_config.sensitivity_x, MIN);
+    else
+        mouse_config.sensitivity_x = std::min(mouse_config.sensitivity_x, -MIN);
+
+    mouse_config.sensitivity_y = std::clamp(sensitivity_y / 100.0f, -MAX, MAX);
+    if (!std::signbit(mouse_config.sensitivity_y))
+        mouse_config.sensitivity_y = std::max(mouse_config.sensitivity_y, MIN);
+    else
+        mouse_config.sensitivity_y = std::min(mouse_config.sensitivity_y, -MIN);
+}
+
+void Mouse_NewScreenParams(uint16_t clip_x, uint16_t clip_y, uint16_t res_x, uint16_t res_y,
+                           bool fullscreen, int32_t x_abs, int32_t y_abs) {
+
+    mouse_video.clip_x     = clip_x;
+    mouse_video.clip_y     = clip_y;
+    mouse_video.res_x      = res_x;
+    mouse_video.res_y      = res_y;
+    mouse_video.fullscreen = fullscreen;
+
+    MouseVMW_NewScreenParams(x_abs, y_abs);
+}
+
+void Mouse_EventMoved(int32_t x_rel, int32_t y_rel, int32_t x_abs, int32_t y_abs, bool is_captured) {
+
+	MouseVMW_NotifyMoved(x_abs, y_abs);
+
+    // TODO: change the code so that in seamless mode sensitivity is not applied
+	Mouse_CursorMoved(x_rel * mouse_config.sensitivity_x,
+					  y_rel * mouse_config.sensitivity_y,
+					  (x_abs - mouse_video.clip_x) / (mouse_video.res_x - 1) * mouse_config.sensitivity_x,
+					  (y_abs - mouse_video.clip_y) / (mouse_video.res_y - 1) * mouse_config.sensitivity_y,
+					  is_captured);
```

Comment:
minor: wonky indentation

Reply:
Used the script to correct all the formatting.